proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -151,6 +151,30 @@ message ResetStickyTaskQueueRequest {
 message ResetStickyTaskQueueResponse {
 }
 
+message ExecuteMultiOperationRequest {
+    string namespace_id = 1;
+    string workflow_id = 2;
+    repeated WorkflowOperation operations = 3;
+}
+
+message WorkflowOperation {
```

Comment:
Why it is not embedded in `ExecuteMultiOperationRequest`?

Reply:
Ah, I missed that one.